tvb_build/deploy_with_keycloak/Jenkinsfile

```diff
@@ -7,8 +7,26 @@ pipeline {
     }
 
     stages {
+        stage('Copy Step 1') {
```

Comment:
This is very good, and will update/maintain on Docker hub thevirtualbrain/tvb-run:keycloak
We should have also thevirtualbrain/tvb-run:latest updated. 
Could it be ok in the same Jenkins job ? 
This part is not part of the current task, rather of TVB-2523, but might be easy to fix here or soon after these changes.

Reply:
I would remove the keycloak tag an keep only the latest one because the differences will be that in the keycloak tag keycloak is enabled by default, but in our deploys (VBC and HBP) we override the tvb config file anyway and we can set this flag to True here.

Reply:
good.